libudpard/CMakeLists.txt

```diff
@@ -0,0 +1,98 @@
+# This software is distributed under the terms of the MIT License.
```

Comment:
nit: Can we call this folder something other than `libudpard` so that it's not confused with the `libudpard` submodule and so that people don't think that this code comes with the official libdupard impementation. Maybe `libudpard_demo`?